rialto_airflow/database.py

```diff
@@ -83,12 +83,12 @@ class Publication(Base):
     pubmed_json = Column(JSONB)
     created_at = Column(DateTime, server_default=utcnow())
     updated_at = Column(DateTime, onupdate=utcnow())
-    authors = relationship(
+    authors: RelationshipProperty = relationship(
```

Comment:
these join tables seem to be something mypy has trouble inferring type on.  now wondering whether it's possible to give this more precision `Author` as the type on the other side.  i only added this because the type checker pointed to this as one of the couple spots that required type clarification.